FirstProject/css/main.css

```diff
@@ -0,0 +1,90 @@
+header {
+    background-image: url(../img/feedback-banner.jpg);
+}
+
+.left_image {
+    height: 290px;
+}
+
+.right_description {
+    font-weight: bold;
+}
+
+
+.container-fluid {
+    background-color: lightgrey;
+}
+
+body {
+    overflow-x:hidden;
+}
+
+a {
+    text-decoration: none!important;
```

Comment:
I used !important to override this style applied on an a. I know that is not recommended but definatelly I will fix this later.